python/ql/test/experimental/query-tests/Security/CWE-295/urlopen.py

```diff
@@ -0,0 +1,26 @@
+import urllib.request
+
+#Simple cases
+urllib.request.urlopen('https://semmle.com', "/path/to/cert/") # BAD (does not assign cert path to "cafile")
+urllib.request.urlopen('https://semmle.com', cafile="/path/to/cert/")
+urllib.request.urlopen('https://semmle.com') # BAD
+urllib.request.urlopen('https://semmle.com', cafile=None) # BAD
+urllib.request.urlopen('http://semmle.com', cafile=None) # OK (not https link)
+
```

Comment:
Could we get an example each with `capath` and `context` (as the query also accounts for these)?